test/.default_test_parameters

```diff
@@ -0,0 +1,4 @@
+tests.build-image-version=0.9.0-rc2
+tests.services-image-version=0.9.0-rc2
```

Comment:
Is there a specific reason for using rc2 as 0.9.0 is released already?

Reply:
jobs-service 0.9.0 does not work...

Reply:
I could set directly the jobs-service 0.9.0-rc2 and let the others to 0.9.0

Reply:
could be good, also please add a comment explaining this setup and when it can be removed

Reply:
updated